persistence/javadsl/src/main/scala/com/lightbend/lagom/javadsl/persistence/PersistentEntity.scala

```diff
@@ -211,8 +211,10 @@ abstract class PersistentEntity[Command, Event, State] {
     private var commandHandlers: Map[Class[_ <: Command], JBiFunction[_ <: Command, CommandContext[Any], Persist[_ <: Event]]] =
       cmdHandlers
 
+    @scala.deprecated
```

Comment:
Add `message` and `since` to the deprecation annotations. For example:

```scala
@deprecated("A message explaining why this method was deprecated", since = "1.5.0")
```

This way, the compiler can give users better warning messages and directions about how to avoid the deprecated APIs.

Reply:
@marcospereira Done! Please let me know for any further changes.